### DISCUSSION TITLE
Section 3.4: Mention and add reference to APE 19 at the end

### DISCUSSION THREAD

**Discussion by aaryapatil** [astropy/astropy-v5.0-paper#102] [Text comments] [answered]
The finance committee recently formulated a set of principles for distributing Project funds in the APE 19 document. We thought it was important enough to be mentioned in the paper but were waiting for it to be approved first. Now that it is accepted and has an associated doi, should we include it? The APE is self-contained and self-explanatory, so we will just have to include a line of description with a reference. Something on the lines of:

> The Project follows a set of principles to fairly and effectively distribute the above described funds. By drawing inspiration from [Numpy's NEP 48](https://numpy.org/neps/nep-0048-spending-project-funds.html), the [APE 19](https://zenodo.org/record/6312048#.YloHt5PMJTZ) document lays out these principles. We refer the reader to that document for more details.

Thoughts?

**Comment by adrn** [comment]
That would be great! Would you mind opening a PR with this added sentence? Thanks!

**Reply by aaryapatil** [reply]
On it!

**Comment by pllim** [comment, marked as answer]
PR is done by @aaryapatil at astropy/astropy-v5.0-paper#103 . Thanks!
